app/actions/remote/thread.ts

```diff
@@ -40,8 +40,7 @@ export const fetchAndSwitchToThread = async (serverUrl: string, rootId: string,
     }
 
     // Load thread before we open to the thread modal
-    // @Todo: https://mattermost.atlassian.net/browse/MM-42232
-    fetchPostThread(serverUrl, rootId);
+    fetchPostThread(serverUrl, rootId, {});
```

Comment:
I wonder if we want to add the options arguments as optional, or we want to be explicit like this.